proto.h

```diff
@@ -92,6 +92,10 @@ int MPI_Alltoallv_sparse(void *sendbuf, int *sendcnts, int *sdispls,
         MPI_Datatype sendtype, void *recvbuf, int *recvcnts,
         int *rdispls, MPI_Datatype recvtype, MPI_Comm comm);
 
+int MPI_Alltoallv_fix_buserror(void *sendbuf, int *sendcnts, int *sdispls,
```

Comment:
Let's call it `MPI_Alltoallv_throttled()` -- we don't know if it fixes the buserror or not, and 2 years from now we won't even remember any details about the bus error on BlueWaters..